Pose2D odometryOffset   # /**< The body motion performed in this step. */
bool isLeavingPossible   # /**< Is leaving the motion module possible now? */
BikeRequest executedBikeRequest  # /**< The bike request that is currently in execution. */
